src/async-components/views/dialogs/security/CreateSecretStorageDialog.tsx

```diff
@@ -585,6 +585,7 @@ export default class CreateSecretStorageDialog extends React.PureComponent<IProp
                     <div>{_t("Enter your account password to confirm the upgrade:")}</div>
                     <div>
                         <Field
+                            id="mx_CreateSecretStorageDialog_password"
```

Comment:
this avoids differences in the jest snapshots depending on how many tests you run